### PR TITLE
Fix local variables that need to be private

### PR DESCRIPTION
**Pull request by ImplicitDeny**
Fix linter warnings for local/private variables.

### CODE REVIEW

**Comment by ImplicitDeny** [comment]
No I don't. These variables cannot be set correctly when using `private`. They need to stay this way for now. It's linked to #21

File: addons/gsri_opex/functions/bays/fn_bayReplace.sqf [hunk 1]
```diff
@@ -1,4 +1,5 @@
 params ["_target", "_player", "_params"];
+private["_type", "_bay", "_cargo"];
```
Comment (ArwynFr) [inline addons/gsri_opex/functions/bays/fn_bayReplace.sqf:2]:
```suggestion
private ["_type", "_bay", "_cargo"];
```

File: addons/gsri_opex/functions/core/fn_initSettings.sqf [hunk 2]
```diff
@@ -6,10 +8,10 @@ _settings = (configProperties [configFile >> "GSRI_FREMM_Templates" >> "GSRI_Nor
 
 // Automatic creation of needed server-forced settings for
 {
-	_template = _x;
+	private _template = _x;
 	{
-		_settingType = ["CHECKBOX","EDITBOX"] select (isText (configFile >> "GSRI_FREMM_Templates" >> _template >> _x));
-		_defaultValue = switch (_settingType) do {
+		private _settingType = ["CHECKBOX","EDITBOX"] select (isText (configFile >> "GSRI_FREMM_Templates" >> _template >> _x));
+		private _defaultValue = switch (_settingType) do {
```
Comment (ArwynFr) [inline addons/gsri_opex/functions/core/fn_initSettings.sqf:14]: Not sure how this works

File: addons/gsri_opex/functions/fremm/fn_fremmAddBridge.sqf [hunk 3]
```diff
@@ -22,7 +23,7 @@ if(isServer) then {
 		_move = selectRandom ["HubSittingAtTableU_idle1", "HubSittingAtTableU_idle2", "HubSittingAtTableU_idle3"];
 		[_unit, _move] remoteExecCall ["switchMove", 0];
 		_unit addEventHandler ["AnimDone", {
-			params ["_u", "_anim"];
+			params ["_u"];
```
Comment (ArwynFr) [inline addons/gsri_opex/functions/fremm/fn_fremmAddBridge.sqf:26]:
```suggestion
			params ["_eventUnit"];
```

File: addons/gsri_opex/functions/fremm/fn_fremmAddBoatBays.sqf [hunk 4]
```diff
@@ -28,17 +28,17 @@ if(isServer) then {
...
+private _actionsList = [];
 {
-	_name = getText (configFile >> "CfgVehicles" >> _x >> "displayName");
-	_actionBoat = [format["action_%1",_x],_name,"",{_this spawn GSRI_fnc_bayReplace},{true},{},[_x]] call ace_interact_menu_fnc_createAction;
+	private _name = getText (configFile >> "CfgVehicles" >> _x >> "displayName");
+	private _actionBoat = [format["action_%1",_x],_name,"",{_this spawn GSRI_fnc_bayReplace},{true},{},[_x]] call ace_interact_menu_fnc_createAction;
 	_actionsList pushBack _actionBoat;
 } forEach ["B_Boat_Transport_01_F", "C_Boat_Transport_02_F", "B_Boat_Armed_01_minigun_F", "B_SDV_01_F"];
-_actionNull = ["actionNull",localize "STR_GSRI_FREMM_emptyBay","",{_this spawn GSRI_fnc_bayReplace},{true},{},[""]] call ace_interact_menu_fnc_createAction;
+private _actionNull = ["actionNull",localize "STR_GSRI_FREMM_emptyBay","",{_this spawn GSRI_fnc_bayReplace},{true},{},[""]] call ace_interact_menu_fnc_createAction;
```
Comment (ArwynFr) [inline addons/gsri_opex/functions/fremm/fn_fremmAddBoatBays.sqf:38]:
```suggestion
_actionsList pushBack ["actionNull",localize "STR_GSRI_FREMM_emptyBay","",{_this spawn GSRI_fnc_bayReplace},{true},{},[""]] call ace_interact_menu_fnc_createAction;
```

File: addons/gsri_opex/functions/core/fn_initSettings.sqf (hunk 2 above)
Comment (ImplicitDeny) [reply addons/gsri_opex/functions/core/fn_initSettings.sqf:14]: What are you unsure of ?

File: addons/gsri_opex/functions/fremm/fn_fremmAddBridge.sqf [hunk 5]
```diff
@@ -22,7 +23,7 @@ if(isServer) then {
...
 			_move = selectRandom ["HubSittingAtTableU_idle1", "HubSittingAtTableU_idle2", "HubSittingAtTableU_idle3"];
 			[_u, _move] remoteExecCall ["switchMove", 0];
```
Comment (ImplicitDeny) [inline addons/gsri_opex/functions/fremm/fn_fremmAddBridge.sqf:28]:
```suggestion
			[_eventUnit, _move] remoteExecCall ["switchMove", 0];
```

File: addons/gsri_opex/functions/core/fn_initSettings.sqf (hunk 2 above)
Comment (ArwynFr) [reply addons/gsri_opex/functions/core/fn_initSettings.sqf:14]: It's just I don't understand what you want to achieve with this code.
Is the switch used to behave depending on the selected template ?

Comment (ImplicitDeny) [reply addons/gsri_opex/functions/core/fn_initSettings.sqf:14]: We need to create different types of CBA settings following what is configured in the template's config :
- for custom textures, name, numbers, which are registered as text values, we need an editbox
- for features like weapons, selectable helicopter, bays, which are registered as numeric booleans, we need a checkbox.
See https://github.com/team-gsri/GSRI-Opex/blob/20de941b7467e42c1136d7841361f9ecbe38560a/addons/gsri_opex/config.cpp#L46 for the configuration of the template

**Pull request merged**